e2e/nomostest/testpredicates/predicates.go

```diff
@@ -1240,3 +1240,36 @@ func ResourceGroupStatusEquals(expected v1alpha1.ResourceGroupStatus) Predicate
 		return nil
 	}
 }
+
+func subjectsEqual(want []string, got []rbacv1.Subject) error {
```

Comment:
This only compares subjects name but not other fields. Suggest renaming to make it more clear.

Reply:
Renamed